docs/specs/build/toc/basics.yml

```diff
@@ -788,3 +788,99 @@ outputs:
           }
        ]
     }
+---
+# 5. Topic href
+inputs:
+  docfx.yml: |
+    content:
+      include:
+        - docs/f1/toc.yml
+  docs/f1/n1.md:
+  docs/f1/index.md:
+  docs/f1/toc.yml: |
+    - name: Normal File Reference 1
+      href: n1.md
+    - name: Normal File Reference 2
+      href: ~/docs/f1/n1.md
+      topichref: ~/docs/f1/index.md
```

Comment:
>topichref [](start = 6, length = 9)

topicHref

Reply:
according to the spec, it should be topichref

---
In reply to: [186667445](https://github.com/dotnet/docfx/pull/2737#discussion_r186667445) [](ancestors = 186667445)

Reply:
which spec? It's "topicHref" in "table-of-contents.md", also in v2.

---
In reply to: [186668617](https://github.com/dotnet/docfx/pull/2737#discussion_r186668617) [](ancestors = 186668617,186667445)

Reply:
my fault, will change

---
In reply to: [186668617](https://github.com/dotnet/docfx/pull/2737#discussion_r186668617) [](ancestors = 186668617,186667445)